vod/mp4/mp4_parser.c

```diff
@@ -533,6 +545,54 @@ mp4_parser_parse_mdhd_atom(atom_info_t* atom_info, metadata_parse_context_t* con
 	return VOD_OK;
 }
 
+static vod_status_t
+mp4_parser_parse_udta_name_atom(atom_info_t* atom_info, metadata_parse_context_t* context)
+{
+	const char* name = (const char*)atom_info->ptr;
+	uint64_t name_len;
+
+	if (name == NULL) {
+		return VOD_OK;
+	}
+
+	// parse the name
+	if ((context->parse_params.parse_type & PARSE_FLAG_UDTA_NAME) == 0)
```

Comment:
for consistency, better reverse the condition and move it to the code that calls this function (see PARSE_FLAG_EDIT_LIST)